python/dask_cudf/dask_cudf/_legacy/io/orc.py

```diff
@@ -7,14 +7,14 @@
 from pyarrow import orc as orc
 
 from dask import dataframe as dd
-from dask.base import tokenize
 from dask.dataframe.io.utils import _get_pyarrow_dtypes
 
 import cudf
 
 
-def _read_orc_stripe(fs, path, stripe, columns, kwargs=None):
+def _read_orc_stripe(input, columns=None, kwargs=None):
     """Pull out specific columns from specific stripe"""
+    fs, path, stripe = input
```

Comment:
Is there a specific reason for packing everything into `input` now or just preference?

Reply:
This is usually just the simplest way to use `from_map`. That said, I can move `fs` into `args=` since it doesn't vary  by partition.